### ISSUE TITLE
detect what's causing NaNs

### ISSUE THREAD

**Issue by thushv89** [thushv89/AdaCNN#25]
Possible causes: 
L2 decay: No
Adaptive dropout: No
Pool Momentum:No
Way params removed:
Data distribution

Currently,
Set use_dropout to False
use_l2_loss to False
pool_momentum = 0.0
rm_indices are a continuous block
** no finetune after remove action ** This seems to be the cause of NaNs

**Comment by thushv89** [comment]
Possibly several causes; "slice_optimize", "learning rate"
